code-challenges.js

```diff
@@ -39,3 +213,34 @@ var numbersToAdd2 = [0, 7, -8, 12]
 // Expected output: [0, 7, -1, 11]
 var numbersToAdd3 = []
 // Expected output: []
+
+
+
+
+//I got this function to add the two previous numbers but had a hard time adding the value of the running sum instead of adding to the previous answers.
+
+
+
+
+//function that takes in array and returns the accumulating sum
+const sumLog = (array) => {
+  let addedArray = array.map((value, index) => {
+    if(index === 0){
+      return value
+    } else{
+//might have to add if statment to address if first value to just drop the value
+    return value + array[index - 1 ] //[index - 1]
+}
+  })
+  return addedArray
+}
+
+
+// const sumLogLoop = (array) => {
+// for(let i = 0; i < array.length); i++)
+// }
+
```

Comment:
I think I see where you are going here. Looks pretty close. But remember value inside of your map is going to reset every time. Consider creating an anchor variable outside your map.